tests/test_covar2d.py

```diff
@@ -75,7 +75,7 @@ def tearDown(self):
     def test01GetMean(self):
         results = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_cov2d_mean.npy'))
         self.mean_coeff = self.cov2d._get_mean(self.coeff_clean)
-        self.assertTrue(np.allclose(results, self.mean_coeff))
+        self.assertTrue(np.allclose(results.flatten(), self.mean_coeff))
```

Comment:
Would be preferable to have `results[:, 0]` here. Explicit is better than implicit.